src/libstd/ffi/c_str.rs

```diff
@@ -271,6 +287,27 @@ impl CString {
     /// to undefined behavior or allocator corruption.
     ///
     /// [`into_raw`]: #method.into_raw
+    ///
+    /// # Examples
+    ///
+    /// Create a `CString`, pass ownership to an `extern` function (via raw pointer), then retake
+    /// ownership with `from_raw`:
+    ///
+    /// ```ignore
```

Comment:
@frewsxcv Currently `no_run` (and `compile_fail` as well) will [stop after analysis](https://github.com/rust-lang/rust/blob/master/src/librustdoc/test.rs#L248-L250). It won't even run codegen.